src/lib/cookies.js

```diff
@@ -107,7 +107,21 @@ class CookieHandler extends CDP {
             await this.Network.deleteCookies(badCookie);
         }
         // Store cookies in the browser
-        await this.Network.setCookies({cookies: browserCookies});
+        const cleanedCookies = browserCookies.reduce((cookies, cookie) => {
+            if (cookie.name === "") return cookies;
+            const cleanedCookie = {...cookie};
+            if (cleanedCookie.path.includes("%2F")) {
```

Comment:
path could be undefined
```suggestion
            if (cleanedCookie.path?.includes("%2F")) {
```